routes.py

```diff
@@ -52,6 +52,8 @@ def graphql_playgroud():
 
 
 @app.route("/graphql", methods=["POST"])
+@cross_origin(origin="localhost", headers=["Content-Type", "Authorization"])
```

Comment:
just a quick note that I'm not sure Chrome supports CORS set to localhost - you may have to use a 'fake' localhost-alternative domain.